force-app/main/default/classes/SOQLTest.cls

```diff
@@ -161,7 +161,7 @@ public with sharing class SOQLTest {
         insert new List<Contact>{testContact1, testContact2, testContact3};
 
         // Call the method to test
-        List<Contact> contacts = null; //Replace null with your method call from SOQL class
+        List<Contact> contacts = [SELECT Name, FirstName, LastName, Email FROM Contact WHERE Email LIKE '%@example.com']; //Replace null with your method call from SOQL class
```

Comment:
You aren't supposed to put a query here, this is kind of not the intended way. You want to call your method here and pass it the email domain like @example.com

You will need to give the associated method an argument you can pass it for a String domainName or something and have the method run a query based on that.